polkadot/runtime/parachains/src/configuration.rs

```diff
@@ -261,6 +261,8 @@ pub struct HostConfiguration<BlockNumber> {
 	/// The minimum number of valid backing statements required to consider a parachain candidate
 	/// backable.
 	pub minimum_backing_votes: u32,
+	/// Node features enablement.
+	pub node_features: NodeFeatures,
```

Comment:
I think a better name is `required_client_capabilities`.

Reply:
I don't think this name is strictly better. I don't have a strong opinion but we've just decided on `node_features`. WDYT @bkchr @alexggh